ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -395,12 +395,12 @@ run :: forall m blk.
     => ChainDBEnv m blk
     ->    Cmd     blk (TestIterator m blk) (TestFollower m blk)
     -> m (Success blk (TestIterator m blk) (TestFollower m blk))
-run env@ChainDBEnv { varDB, .. } cmd =
+run env@ChainDBEnv { args, varDB, .. } cmd =
     readMVar varDB >>= \st@ChainDBState { chainDB = ChainDB{..}, internal } -> case cmd of
       AddBlock blk             -> Point               <$> (advanceAndAdd st (blockSlot blk) blk)
       AddFutureBlock blk s     -> Point               <$> (advanceAndAdd st s               blk)
       GetCurrentChain          -> Chain               <$> atomically getCurrentChain
-      GetLedgerDB              -> LedgerDB . flush    <$> atomically getLedgerDB
+      GetLedgerDB              -> LedgerDB . flush (configSecurityParam $ cdbTopLevelConfig args) <$> atomically getLedgerDB
```

Comment:
This breaks the alignment. Maybe a local binding of `k = configSecurityPram $ cdbToplevelConfig args` can help?